packages/w/waline.json

```diff
@@ -20,10 +20,9 @@
       {
         "basePath": "dist",
         "files": [
-          "*.js"
+          "**/*"
```

Comment:
👋 We try to avoid using globstars as they can very easily lead to abuse of the CDN -- please try to specify what files we're expecting to match based on their extensions and directories.

Reply:
ok, I will add another commit to specify files later.